config/features.yml

```diff
@@ -1790,10 +1790,6 @@ features:
     actor_type: user
     description: Use the original veteran_x models to power Appoint a Rep entity search
     enable_in_development: true
-  va1010_forms_enrollment_system_service_enabled:
-    actor_type: user
-    description: Enables the VA1010Forms enrollment system service
-    enable_in_development: true
   va1010_forms_eesummary_rest_api_enabled:
```

Comment:
Once this is deployed, you'll need to follow step 4 of [this section of this guide](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Removingold/unusedfeaturetoggles) to completely remove this toggle. You'll need to do it in the terminal of each ArgoCD environment. If you or no one on your team has access to those terminals, let me know and I can do it for you. Remember, don't delete them in Argo until the code has been deployed.